packages/www/components/Site/IconGrid/index.tsx

```diff
@@ -41,7 +41,8 @@ export default function IconGrid({ title, richText, icons, portableTextRaw }) {
         <Box
           sx={{
             display: ["block", "flex"],
-            justifyContent: ["unset", "space-between"],
+            flexWrap: "wrap",
+            justifyContent: ["unset", "center"],
```

Comment:
Is this "unset" necessary? I'm not familiar with this array styling feature, would this just set `center` as justify-content?